### PR TITLE
check temp file existence

### PR DESCRIPTION
**Pull request by burak-58**

### CODE REVIEW

File: src/main/java/io/antmedia/muxer/Muxer.java [hunk 1]
```diff
@@ -273,6 +275,9 @@ public void init(IScope scope, final String name, int resolution, boolean overri
 				resourceName += "_" + resolution + "p";
 			}
 
+			
+			System.out.println("rn:"+resourceName+" e:"+extension);
```
Comment (mekya) [inline src/main/java/io/antmedia/muxer/Muxer.java:279]: Use logger instead of sysout

File: src/main/java/io/antmedia/muxer/Muxer.java [hunk 2]
```diff
@@ -286,11 +291,13 @@ public void init(IScope scope, final String name, int resolution, boolean overri
 				if (!overrideIfExist && file.exists()) {
```
Comment (mekya) [inline src/main/java/io/antmedia/muxer/Muxer.java:291]: I think it should check the tmp file even here. as well for the first recording

Comment (burak-58) [reply src/main/java/io/antmedia/muxer/Muxer.java]: also checked and test added

File: src/main/java/io/antmedia/muxer/Muxer.java (hunk 1 above)
Comment (burak-58) [reply src/main/java/io/antmedia/muxer/Muxer.java]: removed

File: src/main/java/io/antmedia/muxer/MuxAdaptor.java [hunk 3]
```diff
@@ -379,17 +379,22 @@ public boolean init(IScope scope, String streamId, boolean isAppend) {
...
+		enableWebMSetting();
 		initVertx();
 
 		if (mp4MuxingEnabled) {
 			addMp4Muxer();
 			logger.info("adding MP4 Muxer, add datetime to file name {}", addDateTimeToMp4FileName);
 		}
+		
+		if (webMMuxingEnabled) {
+			addWebMMuxer();
```
Comment (mekya) [inline src/main/java/io/antmedia/muxer/MuxAdaptor.java:395]: If it'a H264 and AAC stream, adding webm muxer causes problems?

**Review by mekya — APPROVED** [review]

**Pull request merged**